DESIGN.md

```diff
@@ -1,4 +1,73 @@
 # PowerShell Manager: Design and Architecture
 
-<!-- TODO: Fill this out! -->
-<!-- NOTE: Not multi-thread safe, executions against the same manager in parallel can get _weird_ -->
\ No newline at end of file
+<!-- NOTE: Not multi-thread safe, executions against the same manager in parallel can get _weird_ -->
+
+Managing Windows infrastructure from ruby is difficult - much of the interaction with the tools and APIs is performed via PowerShell, and PowerShell remains the most common language for operators of Windows machines to know.
```

Comment:
ruby or Ruby?

Reply:
If you refer to PowerShell as cased, then Ruby gets the same treatement

Reply:
We shouldn't start the document with a negative, and it's not the point of this gem to manage windows from ruby, its to make using powershell easier from ruby. We don't have to justify using powershell on windows, we have to justify using our gem.

This gem reduces the difficulty of running PowerShell in Ruby by X, Y, and Z.

Reply:
Resolved with a substantial reword, I think.